app-ingest/src/test/scala/com/azavea/rf/JobDefinitionSpec.scala

```diff
@@ -0,0 +1,14 @@
+package com.azavea.rf
+
+import spray.json._
+import org.scalatest._
+
+class JobDefinitionSpec extends WordSpec with Matchers {
+  "JobDefinitions" should {
+    "be json-readable and writable" in {
+      val json = """{"scenes":[{"output":"s3://raster-foundary/userName/layerName","source":{"extent":[0,0,10,10],"urls":["s3://bucket/landsat/LC8_file1.tif","s3://bucket/landsat/LC8_file2.tif","s3://bucket/landsat/LC8_file3.tif"]}}]}""".parseJson
```

Comment:
Once this is stabilized, it would be good to document this somewhere since this is what the airflow import jobs are going to need to produce.